dev/com.ibm.ws.jpa_22_fat/publish/servers/JPADefaultDataSourceServer_JTA/server.xml

```diff
@@ -0,0 +1,48 @@
+<!--
+    Copyright (c) 2019 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="JPA Default DataSource Server">
+
+	<featureManager>
+		<feature>servlet-4.0</feature>
+		<feature>jpa-2.2</feature>
+	    <feature>componenttest-1.0</feature>
+    </featureManager>
+    
+    <include location="../fatTestPorts.xml"/>
+    
+    <jpa defaultJtaDataSourceJndiName="jdbc/JTA_DS1" />
+    
+    <dataSource id="jdbc/JTA_DS1" jndiName="jdbc/JTA_DS1" fat.modify="false">
```

Comment:
probably don't need `fat.modify="true"` on any of these config elements. I don't think these tests would benefit from DB rotation testing in the future.
Same comment applies to other server.xml configs added also

Reply:
Done